qdrant-landing/content/demo/demo-4.md

```diff
@@ -0,0 +1,10 @@
+---
+draft: false
+title: Semantic code search
+short_description: Searching over Qdrant source code using semantic search
+description: Navigating an unknown codebase can be a daunting task. This demo shows how to implement a semantic search application for code search tasks, using two different neural encoders. The demo uses a general-purpose sentence transformer and a code-specific model to allow both natural and code-like queries and cover a broader spectrum of possible interactions.
```

Comment:
```suggestion
description: It can be difficult to go through an unknown codebase. This demo shows how to implement a semantic search application for code search tasks, with two neural encoders. These encoders are a general-purpose sentence transformer and a code-specific model. This supports both natural and code-like queries, which covers a broad range of interactions.
```